react-frontend/src/css/global.css

```diff
@@ -0,0 +1,7 @@
+a {
+  color: #1a5a96;
```

Comment:
I'd love if we abstracted colours into a theme over just static css. I think this is something we'd want to discuss as a team. There are a ton of new tech thats been developed to tackle css in js that work really really well with how UX designers develop their mockups and designs

Reply:
in principle I agree, but are you okay with this coming in as is right now?  It's purpose is to fix the url link colours on the site

Reply:
yes im okay with it. I'd push for something like emotion themeing shortly tho

Reply:
Sweet if you've started the global.css I can work to start compiling a lot of stuff